io/io/src/RRawFile.cxx

```diff
@@ -208,6 +211,41 @@ size_t ROOT::Internal::RRawFile::ReadAt(void *buffer, size_t nbytes, std::uint64
    return totalBytes;
 }
 
+void ROOT::Internal::RRawFile::PossiblyInjectFailure(void* buffer, size_t total_bytes, std::uint64_t offset)
+{
+   // check probability of error occurring here
+
+   // trigger particular failure
+   switch(GetFailureInjectionParams().failureType) {
+      case 0:
+         std::cout << "Triggering Bit Flip" << std::endl;
+         //TriggerBitFlip(buffer, total_bytes, offset);
```

Comment:
I cannot see the implementation of `TriggerBitFlip` / `TriggerShortRead` here.  Did you submit the latest changes as part of the PR?